README.md

```diff
@@ -145,22 +145,23 @@ Self-hosted
+| [dart-lnurl](https://github.com/saentari/dart_lnurl)                 | [01][01] [02][02] [03][03] [04][04] [05][05] [06][06] [09][09] [10][10] [11][11] [12][12] [14][14] [17][17] [18][18] [19][19]                            |
+| [go-lnurl](https://github.com/fiatjaf/go-lnurl)                      | [01][01] [02][02] [03][03] [04][04] [06][06] [08][08] [09][09] [10][10] [11][11] [12][12] [14][14] [16][16] [18][18] [20][20]                            |
+| [js-lnurl](https://github.com/fiatjaf/js-lnurl)                      | [01][01] [02][02] [03][03] [04][04] [06][06] [08][08] [09][09] [10][10] [11][11] [12][12]                                                                |
+| [lnurl-node](https://github.com/chill117/lnurl-node)                 | [01][01] [02][02] [03][03] [04][04] [06][06] [08][08] [09][09] [10][10] [11][11] [12][12]                                                                |
+| [lnurl-platformio][platformio]                                       | [01][01] [03][03]                                                                                                                                        |
+| [lnurl-ruby](https://github.com/bumi/lnurl-ruby)                     | [01][01] [06][06] [09][09]                                                                                                                               |
+| [lnurlauth](https://github.com/xplorfin/lnurlauth)                   | [01][01] [04][04]                                                                                                                                        |
+| [php-lnurl](https://github.com/tkijewski/php-lnurl)                  | [01][01]                                                                                                                                                 |
+| [python-lnurl](https://github.com/python-ln/lnurl)                   | [01][01] [02][02] [03][03] [04][04] [06][06]                                                                                                             |
+| [bitcoin-s](https://github.com/bitcoin-s/bitcoin-s)                  | [01][01] [03][03] [06][06] [09][09]                                                                                                                      |
+| [rust-lnurl][rust-lnurl]                                             | [01][01] [03][03] [04][04]                                                                                                                               |
+| [spring-lnurl][springlnurl]                                          | [01][01] [04][04]                                                                                                                                        |
+| [lnurl_client Elixir][https://github.com/ramontayag/lnurl_client-ex] | [01][01] [06][06]                                                                                                                                        |
```

Comment:
This link is broken

```suggestion
| [lnurl_client Elixir](https://github.com/ramontayag/lnurl_client-ex) | [01][01] [06][06]                                                                                                                                        |
```

Reply:
🤦 Sorry, didn't check! Will fix along with the [request](https://github.com/lnurl/luds/pull/188#issuecomment-1310072586) of @fiatjaf

Reply:
Done - tested it on my branch as well.

Reply:
Ah yes at the bottom is much better 🙏